modules/ossm-rn-known-issues.adoc

```diff
@@ -198,6 +201,14 @@ New issues for Kiali should be created in the link:https://issues.redhat.com/pro
 
 These are the known issues in Kiali:
 
+* https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click on the **Node graph** menu option of the traffic graph menu of any node, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. There is currently no workaround for this issue.
```

Comment:
This is not quite clear to me. I think it is better: 

```suggestion
* https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click on the **Node graph** option of any node context menu within the traffic graph, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. There is currently no workaround for this issue.
```

This is a screenshot of the page:
![image](https://github.com/openshift/openshift-docs/assets/122779323/7387ddb4-fc9d-43ac-b7ca-24d5da6d4e23)